### PR TITLE
.gitignore and npm command updated

### PR DESCRIPTION
**Pull request by lavish205**
#358

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /.venv
 /dist
+./dist
```
Comment (n1k0) [inline .gitignore]: So this was already setup as expected, this is unneeded.

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "dist-fx": "BABEL_ENV=firefox browserify -s loadKinto --ignore isomorphic-fetch -e fx-src/index.js -o temp.jsm -t [ babelify --sourceMapRelative . ] && mkdir -p dist && cp fx-src/jsm_prefix.js dist/moz-kinto-client.js && echo \"\n/*\n * Version $npm_package_version - $(git rev-parse --short HEAD)\n */\n\" >> dist/moz-kinto-client.js && cat temp.jsm >> dist/moz-kinto-client.js && rm temp.jsm",
     "compute-sri": "cd dist; for file in $(ls kinto-*.js); do printf \"| %-23s | %-64s |\\n\" ${file} $(echo -n 'sha384-' && cat ${file} | openssl dgst -sha384 -binary | openssl enc -base64); done",
     "publish-demo": "npm run dist-prod && cp dist/kinto-$npm_package_version.js demo/kinto.js && gh-pages -d demo",
+    "publish-to-npm": "npm run build && npm publish",
```
Comment (n1k0) [inline package.json]: So here we still need to build the dist files, which isn't covered by the sole build command. Add `npm run dist` to the chain.

**Comment by n1k0** [comment]
LGTM, thanks!

**Comment by n1k0** [comment]
Dunno why no build is performed by travis. We're unable to land this until we at least get a green pill...

**Comment by lavish205** [comment]
Don't know why it hasn't started yet. Any idea @leplatrem @Natim @n1k0

**Comment by n1k0** [comment]
Oh well. Landed.

**Pull request merged**